packages/cozy-notifications/README.md

```diff
@@ -0,0 +1,190 @@
+`cozy-notifications` provides tools to send notifications (push or email) from a Cozy
+application or konnector.
+
+## `sendNotification`
+
+The main entrypoint of this library is `sendNotifification(cozyClient, notificationView)`.
+
+Before being able to use it, you must define a notification view class.
+
+## Notification views
+
+Notification views are responsible for
+
+- fetching the data necessary for the notification
+- providing a template for the notification
+- configuring the notification (`category`, `channels`)
+- deciding if the notification should be sent
+
+```
+class MyNotification {
+  async buildData() {
+    return {
+      // data that will be used in the template
+    }
+  }
+
+  getPushContent() {
+    return 'push content'
```

Comment:
can it be also an html template? Or should it be only a string? TBH I really see how to use this lib to send an email, but not to send a push notif

Reply:
Yes, I think the doc can be improved on this point. The title used for the notification will be the result of getTitle(), and the content will be the result of getPushContent().

Reply:
Done